libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -220,4 +224,18 @@ export class MessageRepository extends BaseRepository<MessageEntity> {
       data: this.mapEntities(response),
     };
   }
+
+  async delete(query: FilterQuery<MessageEntity & Document>) {
+    const message = await this.findOne({ _id: query._id });
+    if (!message) {
+      throw new DalException(`Could not find feed with id ${query._id}`);
```

Comment:
Small suggestion I think there is a typo
```suggestion
      throw new DalException(`Could not find a message with id ${query._id}`);
```